### PR TITLE
News section

### PR DESCRIPTION
**Pull request by MZeshanShoukat**

### CODE REVIEW

**Comment by alydersen** [comment]
@MZeshanShoukat The header of this section behaves strange on window resize (top margin to small):
image

File: i18n/en/code.json [hunk 1]
```diff
@@ -654,5 +654,21 @@
 "home.event.keypoints_desc3": {
   "message": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
   "description": "The description of third keypoint"
-}
+},
```
Comment (alydersen) [inline i18n/en/code.json:657]: What are line 657 to 669 doing?

File: i18n/nb/code.json [hunk 2]
```diff
@@ -643,5 +643,21 @@
 "home.event.keypoints_desc3": {
   "message": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
   "description": "The description of third keypoint"
-}
+},
```
Comment (alydersen) [inline i18n/nb/code.json:646]: What are line 647 to 658 doing?

**Comment by alydersen** [comment]
@CathrineDQ What is this? 
image

**Comment by MZeshanShoukat** [comment]
> @MZeshanShoukat The header of this section behaves strange on window resize (top margin to small): image


Updated now

**Comment by MZeshanShoukat** [comment]
> @CathrineDQ What is this? image


Requirement was that we need to create a new dropdown in the header menu for show the option of contact us and book a meeting

**Comment by CathrineDQ** [comment]
@MZeshanShoukat @alydersen 
Regarding the contact us button (the image) - the idea was to create the contact button where you can choose either to go to "book a meeting" or send a message (via the mailchimp link). However, I would prefer that it was "floating" in the lower right corner, not at the top.

**Comment by MZeshanShoukat** [comment]
> @MZeshanShoukat @alydersen Regarding the contact us button (the image) - the idea was to create the contact button where you can choose either to go to "book a meeting" or send a message (via the mailchimp link). However, I would prefer that it was "floating" in the lower right corner, not at the top.

Update the button accordingly

**Comment by alydersen** [comment]
Nice work @MZeshanShoukat and @CathrineDQ! Merging now

**Review by alydersen — APPROVED** [review]

**Pull request merged**